src/app/components/app.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Toolbar } from './toolbar';
+import { Navbar } from './navbar';
+import { GithubPage } from '../../github';
+
+export default (props) => (
+    <div>
+        <Toolbar />
+        <Navbar />
+        <GithubPage />
+        {props.children}
```

Comment:
`{props.children}` is where the route gets rendered. So no need for specifying the `<GithubPage />` here